registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -5031,6 +5031,103 @@
           "tokenManagerType": "mintBurn"
         }
       ]
+    },
+    "0xa7457b77fed88990d9b1d8a222d99fc8d018b651f5085236ba5f5ef425baeb58": {
+      "tokenId": "0xa7457b77fed88990d9b1d8a222d99fc8d018b651f5085236ba5f5ef425baeb58",
+      "deployer": "0xfBe09B7A6C6cc3A72B4CA41cdD9c88AC62bb7937",
+      "originalMinter": null,
+      "prettySymbol": "BUNNI",
+      "decimals": 18,
+      "originAxelarChainId": "Ethereum",
+      "tokenType": "customInterchain",
+      "deploySalt": "0xf00000000000000000000000001b71977417651eae98854a09fef1b1a02a0f62",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/bunni.svg"
+      },
+      "coinGeckoId": "bunni",
+      "deploymentMessageId": "",
+      "chains": [
+        {
+          "symbol": "BUNNI",
+          "name": "Bunni",
+          "axelarChainId": "Ethereum",
+          "tokenAddress": "0x000000C396558ffbAB5Ea628f39658Bdf61345b3",
+          "tokenManager": "0xFF4719131C34d8913abE4Ff728f29498ab911aB1",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "BUNNI",
+          "name": "Bunni",
+          "axelarChainId": "arbitrum",
+          "tokenAddress": "0x000000C396558ffbAB5Ea628f39658Bdf61345b3",
+          "tokenManager": "0xFF4719131C34d8913abE4Ff728f29498ab911aB1",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "BUNNI",
+          "name": "Bunni",
+          "axelarChainId": "base",
+          "tokenAddress": "0x000000C396558ffbAB5Ea628f39658Bdf61345b3",
+          "tokenManager": "0xFF4719131C34d8913abE4Ff728f29498ab911aB1",
+          "tokenManagerType": "mintBurn"
+        },
+        {
+          "symbol": "BUNNI",
+          "name": "Bunni",
+          "axelarChainId": "binance",
+          "tokenAddress": "0x000000C396558ffbAB5Ea628f39658Bdf61345b3",
+          "tokenManager": "0xFF4719131C34d8913abE4Ff728f29498ab911aB1",
```

Comment:
This doesn't exist. Can you check it? See [this](https://bscscan.com/address/0xFF4719131C34d8913abE4Ff728f29498ab911aB1).